documentation/writing-policies.md

```diff
@@ -131,5 +131,14 @@ Operators supported:
 - Equal
 - NotEqual
 
+# Auto generating rules for pod controllers
+A pod policy is automatically applied to pod controllers. By default, Kyverno inserts an annotation `pod-policies.kyverno.io/autogen-controllers=all`, to generate an additional rule that is applied to pod controllers: DaemonSet, Deployment, Job, StatefulSet.
```

Comment:
Suggest changing `A pod policy is automatically applied to pod controllers` to:

````
Writing policies on pods helps address all pod creation flows, but results in errors not being reported when a pod controller object is created. Kyverno solves this issue, by automatically generating rules for pod controllers from a rule written for a pod. 

This behavior is controlled by the  pod-policies.kyverno.io/autogen-controllers annotation. By default, ....

````